extras/data/JLABEG1DVCS/metadata.yaml

```diff
@@ -0,0 +1,23 @@
+nnpdf_id: null
+fktable_id: null
+hepdata: https://www.hepdata.net/record/ins1292133
+
+JLABEG1DVCS_NC_5GEV_ED_F1:
+  description: unpolarized structure function $F_1$ in JLABEG1DVCS with deuteron target
```

Comment:
DVCS is not part of the experiment, please adjust the description to reflect that

Reply:
I'm not sure about this: again the collaboration was called CLAS and at page 3 of https://arxiv.org/pdf/1404.6231.pdf they actually say `The experiment, named “eg1-dvcs” ...` where `dvcs` stands for Deeply Virtual Compton Scattering.

Reply:
well, if the experimentalist say so - fine.